psalm.xml.dist

```diff
@@ -5,6 +5,7 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+    autoloader="bootstrap/autoload.php"
```

Comment:
How about just `vendor/autoload.php`

Reply:
Still working on it, thinking how to manage.